lib/cli/commands/orderbook.ts

```diff
@@ -0,0 +1,183 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient  } from '../command';
+import { ListOrdersRequest, ListOrdersResponse, Order } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { SATOSHIS_PER_COIN } from '../utils';
+
+type FormattedOrderbook = {
+  pairId: string,
+  rows: string[][],
+};
+
+type PriceBucket = {
+  price: number,
+  depth: number;
+};
+
+type OrderbookJson = {
+  pairId: string,
+  sell: PriceBucket[],
+  buy: PriceBucket[],
+};
+
+const COLUMNS = [19, 19, 19, 19];
+const COLUMNS_IN_ORDER_SIDE = COLUMNS.length / 2;
+const HEADER = [
+  { content: colors.green('Buy'), colSpan: 2 },
+  { content: colors.red('Sell'), colSpan: 2 },
+];
+const SECONDARY_HEADER = [
+  colors.green('Price'),
+  colors.green('Depth'),
+  colors.red('Price'),
+  colors.red('Depth'),
+];
+
+const addSide = (buckets: PriceBucket[], precision: number): string[] => {
+  const bucket = buckets.pop();
+  if (bucket) {
+    return [
+      bucket.price.toString(),
+      (bucket.depth / SATOSHIS_PER_COIN).toFixed(precision),
+    ];
+  } else {
+    return Array.from(Array(COLUMNS_IN_ORDER_SIDE)).map(() => '');
+  }
+};
+
+export const createOrderbook = (orders: ListOrdersResponse.AsObject, precision: number) => {
+  const formattedOrderbooks: FormattedOrderbook[] = [];
+  orders.ordersMap.forEach((tradingPair) => {
+    const buy = createOrderbookSide(tradingPair[1].buyOrdersList, false, precision);
+    const sell = createOrderbookSide(tradingPair[1].sellOrdersList, true, precision);
+    const totalRows = buy.length < sell.length
+      ? sell.length : buy.length;
+    const orderbookRows = Array.from(Array(totalRows))
+      .map(() => {
+        return addSide(buy, precision).concat(addSide(sell, precision));
+      });
+    formattedOrderbooks.push({
+      pairId: tradingPair[0],
+      rows: orderbookRows,
+    });
+  });
+  return formattedOrderbooks;
+};
+
+const createTable = () => {
+  const table = new Table({
+    colWidths: COLUMNS,
+  }) as HorizontalTable;
+  table.push(HEADER);
+  table.push(SECONDARY_HEADER);
+  return table;
+};
+
+const displayOrderbook = (orderbook: FormattedOrderbook) => {
+  const table = createTable();
+  orderbook.rows.forEach(row => table.push(row));
+  console.log(colors.underline(colors.bold(`\nTrading pair: ${orderbook.pairId}`)));
+  console.log(table.toString());
+};
+
+const displayTables = (orders: ListOrdersResponse.AsObject, argv: Arguments) => {
+  createOrderbook(orders, argv.precision).forEach(displayOrderbook);
+};
+
+const getAvailablePrices = (orders: Order.AsObject[], sellSide: boolean, count = 8): number[] => {
+  const sortedOrders = [
```

Comment:
I believe xud is returning the orders sorted now, so no need to sort them here.

Reply:
Should be changed if it's indeed already sorted